web/frontend/package.json

```diff
@@ -37,7 +37,7 @@
     "@improbable-eng/grpc-web": ">=0.15",
     "@viamrobotics/prime": ">=0.5",
     "@viamrobotics/rpc": ">=0.2",
-    "@viamrobotics/sdk": "0.12.0-rc.0",
+    "@viamrobotics/sdk": "0.12.1",
```

Comment:
SDK needs to be bumped in RDK outside of this PR. Will remove this once that's done.

Reply:
I think I was wrong about this.

@ethanlook any reasons not to bump this in this PR?

Reply:
No reason! Please go ahead.